### PR TITLE
backend and frontend are added for rhyme

### PR DESCRIPTION
**Pull request by hsnbsrbalaban**

### CODE REVIEW

File: practice-app/backend/api/Api.js [hunk 1]
```diff
@@ -53,11 +53,21 @@ router.use('/definition',function(req,res){
...
   }
   words.examples(db,req,res);
 });
+
+router.use('/rhymes',function(req,res){
+  if(!db.isConnected()){
+    res.send("9");
+    return;
+  }
+  user.rhymes(db,req,res);
```
Comment (fahricansanli) [inline practice-app/backend/api/Api.js]: This should be words.ryhmes

File: practice-app/frontend/yallp/src/app/model.ts [hunk 2]
```diff
@@ -43,3 +43,8 @@ export interface ExampleModel{
     word: string,
     examples: [string]
 };
+
+export interface RhymeModel{
+    word: string,
+    examples: [string]
```
Comment (fahricansanli) [inline practice-app/frontend/yallp/src/app/model.ts]: Are we sure RyhmeModel contains a field named Examples? Or should it be ryhmes?

File: practice-app/frontend/yallp/src/app/property/property.component.html [hunk 3]
```diff
@@ -31,16 +31,19 @@
                 <tr *ngIf="property=='example'">
                   <th>Examples</th>
                 </tr>
+                <tr *ngIf="property=='rhyme'">
+                  <th>Examples</th>
```
Comment (fahricansanli) [inline practice-app/frontend/yallp/src/app/property/property.component.html]: I think this line should be Ryhmes not Examples

File: practice-app/frontend/yallp/src/app/property/property.component.ts [hunk 4]
```diff
@@ -74,5 +74,10 @@ export class PropertyComponent implements OnInit {
         this.data = data.examples;
       });
     }
+    else if(this.property=="rhyme"){
+      this.api.getRhyme(this.searchWord).pipe(filter( data => Object.keys(data).length > 0),take(1)).subscribe(async (data:RhymeModel) => {
+        this.data = data.rhymes;
```
Comment (fahricansanli) [inline practice-app/frontend/yallp/src/app/property/property.component.ts]: You should assign data.ryhmes.all to data

**Comment by muratbuldu** [comment]
The bugs are fixed and ready to merge.

**Pull request merged**